EIPS/eip-2569.md

```diff
@@ -34,32 +34,33 @@ Although this is an applicable way to display an image for a token, the image is
 
 For a fungible token such as an ERC-20 token no such an ID is needed and our proposed interface just assigns an SVG image's file content to a string variable of the token's contract.
 
-```imageSvg```: we use a string variable to save an SVG image file's content.
+`imageSvg`: we use a string variable to save an SVG image file's content.
 
 An SVG image that will be saved in the imageSvg string should include at least two attributes:"name", "desc"(description).
 
-```setTokenImageSvg```: for an ERC-721 or ERC-1155 token or a token implemented by a contract which has a member "ID" to specify its token type or token index we define an interface to associate an SVG image to the token's ID number. For an ERC-20 token or a token implemented by a contract which doesn't have a member "ID" to specify its token type or token index we define an interface to assign an SVG image to a member variable string of this token's contract.
+`setTokenImageSvg`: for an ERC-721 or ERC-1155 token or a token implemented by a contract which has a member "ID" to specify its token type or token index we define an interface to associate an SVG image to the token's ID number. For an ERC-20 token or a token implemented by a contract which doesn't have a member "ID" to specify its token type or token index we define an interface to assign an SVG image to a member variable string of this token's contract.
 
-```getTokenImageSvg```: for an ERC-721 or ERC-1155 token or a token implemented by a contract which has a member "ID" to specify its token type or token index we define an interface to get an SVG image by using the token's ID number. For an ERC-20 token or a token implemented by a contract which doesn't have a member "ID" to specify its token type or token index we define an interface to get an SVG image for it if the token has a member variable string to save the image.
+`getTokenImageSvg`: for an ERC-721 or ERC-1155 token or a token implemented by a contract which has a member "ID" to specify its token type or token index we define an interface to get an SVG image by using the token's ID number. For an ERC-20 token or a token implemented by a contract which doesn't have a member "ID" to specify its token type or token index we define an interface to get an SVG image for it if the token has a member variable string to save the image.
 
 
 ## Specification
-Our proposed interfaces define how a smart contract saves an image for a universal token in Ethereum which keeps the image permanent and tamper-resistant, and how a smart contract retrieves an image from Ethereum for a universal token.  
 
-By calling our proposed interfaces we assume users are accessing an SVG image. 
+An EIP-2569 compatible contract MUST have a method with the signature getTokenImageSvg(uint256) view returns (string memory) and a method with the signature setTokenImageSvg(uint256 tokenId, string memory imagesvg) internal. 
```

Comment:
> `setTokenImageSvg(uint256 tokenId, string memory imagesvg) internal`

Internal functions aren't generally part of a standard.  Only publicly visible functions and functionality should ever be standardized.